chunky/src/java/se/llbit/chunky/entity/StandingBanner.java

```diff
@@ -468,6 +468,6 @@ public static Material getBannerTexture(JsonObject design) {
     }
     Texture texture = new Texture(tinted);
     TextureCache.put(design, texture);
-    return new TextureMaterial(texture);
+    return TextureMaterial.getForTexture(texture);
```

Comment:
The texture is created two lines further up and will always be different and can't be cached this way.

Reply:
You're right, I replaced without thinking about it